### DISCUSSION TITLE
Add additional option to create Dockerfile and docker-compose

### DISCUSSION THREAD

**Discussion by atrakic** [microsoft/aspire#2485] [Q&A] [answered]
During project bootstrap, add additional option to create Dockerfile, docker-compose.yml files, eg.:

```
$ dotnet new aspire-starter --use-redis-cache --output AspireSample --use-containers 
```

**Comment by davidfowl** [comment, marked as answer]
The aspire host is the orchestrator, I'm not sure why we would create docker compose or docker files for these projects.
